### PR TITLE
Issue #1163 - Convert RubiksCubePatterns to use templates

### PR DESCRIPTION
**Pull request by Mailkov**
-----
IA Page: https://duck.co/ia/view/rubiks_cube_patterns

### CODE REVIEW

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=rcube stripes)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/rubiks_cube_patterns_mobile.png)

**Comment by moollaza** [comment]
@Mailkov thanks!

(nevermind the above screenshot, it used the main site instead of duckduckgo -- we're fixing that...)

I think, for the cases where there's only 1 line of output, we should use the text template and use a title + subtitle.

For multiple lines of output, we can stick to the list template. It would be nice to have a title that gives some context about what the user is seeing.

---

Also, I made a typo and still got a result: https://beta.duckduckgo.com/?q=rcube+in+a+cuve&ia=answer

Seems like it spits out all possibilities? We should fix the handling of that so it doesn't return anything

File: lib/DDG/Goodie/RubiksCubePatterns.pm [hunk 1]
```diff
@@ -53,34 +54,48 @@ sub render_text($) {
...
-	#display the cheatsheet
-	my $output = my $html_output = "";
-	foreach my $pattern (keys %patterns) {
-		$output .= render_text($pattern);
-		$html_output .= render_html($pattern);
-	}
-
-	return $output, html => $html_output, heading => "Rubik's Cube Patterns";
+	
+	my %patterns_answer;
```
Comment (mintsoft) [inline lib/DDG/Goodie/RubiksCubePatterns.pm]: @Mailkov Can you fix the indentation here? It should be 4 spaces/tab

Comment (Mailkov) [inline lib/DDG/Goodie/RubiksCubePatterns.pm]: @mintsoft sure ! :+1:

**Comment by mintsoft** [comment]
Excellent, this LGTM. Thanks again @Mailkov

**Pull request merged**